tests/personal_data/positions/test_position.py

```diff
@@ -381,3 +466,692 @@ async def test_update_size_from_order_with_short_overbought_hedge_position(futur
     assert position_inst.size == constants.ZERO
     assert position_inst.is_idle()
 
+
+async def test__calculates_size_update_from_filled_order_long_increasing_one_way(
+        future_trader_simulator_with_default_linear):
+    config, exchange_manager_inst, trader_inst, default_contract = future_trader_simulator_with_default_linear
+    if os.getenv('CYTHON_IGNORE'):
```

Comment:
should we put those at the end of the test, right before when there is an issue ? So that we know why they are here. Or it's on `exchange_manager_inst.exchange.set_pair_future_contract` that it's a problem ?

Reply:
Why not